src/d/alexa-openwebif/source/intents/togglemute.d

```diff
@@ -20,9 +22,18 @@ final class IntentToggleMute : BaseIntent
 	///
 	override AlexaResult onIntent(AlexaEvent, AlexaContext)
 	{
-		immutable res = apiClient.vol("mute");
-
 		AlexaResult result;
+		Vol res;
+		try
+		{
+			res = apiClient.vol("mute");
+		}
+		catch (Exception e)
+		{
+			result = returnError(this);
```

Comment:
It is planned to extend these `catch()` blocks with `stderr` outputs which should contain information of the `returnError()` function. So the step is neccessary. I've opened #76 for this